web/client/translations/data.nl-NL.json

```diff
@@ -1413,6 +1413,7 @@
             "showPreview": "Voorbeeld weergeven",
             "advancedSettings": "Geavanceerde instellingen",
             "templateMetadataAvailable": "Metadata beschikbaar in Dublin Core-formaat: abstract, boundingBox, bijdrager, maker, beschrijving, formaat, identifier, referenties, rechten, bron, onderwerp, tijdelijk, titel, type, uri",
+            "invalidUrlHttpProtocol": "De URL komt niet overeen met het MapStore HTTP-protocol {protocol}",
```

Comment:
We usually translate only the default languages supported by mapstore DE, US, ES, FR and IT so this could be removed